src/Shared/ServerInfrastructure/StringUtilities.cs

```diff
@@ -769,7 +769,7 @@ private static void PopulateSpanWithHexSuffix(Span<char> buffer, (string? str, c
             // This must be explicity typed as ReadOnlySpan<byte>
             // This then becomes a non-allocating mapping to the data section of the assembly.
             // If it is a var, Span<byte> or byte[], it allocates the byte array per call.
-            ReadOnlySpan<byte> hexEncodeMap = new byte[] { (byte)'0', (byte)'1', (byte)'2', (byte)'3', (byte)'4', (byte)'5', (byte)'6', (byte)'7', (byte)'8', (byte)'9', (byte)'A', (byte)'B', (byte)'C', (byte)'D', (byte)'E', (byte)'F' };
+            ReadOnlySpan<byte> hexEncodeMap = "0123456789ABCDEF";
```

Comment:
Remember the u8 suffix here, otherwise it'll break again in the next preview.

Reply:
Whoops, thanks for catching that